tasks/auditd.yml

```diff
@@ -10,6 +10,8 @@
     owner: root
     group: root
   when: ansible_os_family == "Debian"
+  notify:
+    - Update GRUB
```

Comment:
Why is this necessary?

Reply:
I see that this was done as part of commit 452e5c9aecf680b6436ccca439f857173c691cdf, where you are updating `grub` anytime the `grub` configuration is modified.  Can you add a comment to that effect?

Reply:
Taken care of in commit 6e19020.